apps/settings/style/lists.css

```diff
@@ -18,6 +18,13 @@ ul li {
   color: #000;
 }
 
+/* Provide additional margin around gaia-switch to match spacing with legacy components */
+ul li > gaia-switch {
+  -moz-margin-start: 1.5rem;
+  /* The !important is necessary to override gaia-swtich scoped style. */
+  width: calc(100% - 1.5rem) !important;
+}
```

Comment:
For now I've updated the component color, and in the future this should move to gaia-theme.

https://github.com/KevinGrandon/gaia/blob/bug_1168860_settings_display_panel_gaia_switch/shared/elements/gaia_switch/style.css#L22